### PR TITLE
Update documentation for 4.5

### PR DESCRIPTION
**Pull request by keithf4**
# Description  

Update documentation for version 4.5

Include updates to hugo theme and new shortcodes

Fixed incorrect variable name and inconsistency with using the cluster_name variable in the Prometheus target pg example files.

## Type of change  
Please check all options that are relevant  
- [x] Bug fix (change which fixes an issue)  
- [ ] Feature (change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [x] Documentation update only  

# How Has This Been Tested?  

**Tested Configuration**:  
- [ ] CentOS, Specify version(s):  
- [ ] PostgreQL, Specify version(s):  
- [ ] docs tested with hugo <0.60  

Tested with playbook(s):  
# Checklist:  
- I have made corresponding changes to:  
    - [ ] the documentation  
    - [ ] the release notes  
    - [ ] the upgrade doc

### CODE REVIEW

File: hugo/content/prometheus/_index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ pgMonitor assumes to be using at least Prometheus 2.9.x. We recommend to always
 
 ##### User and Configuration Directory Installation
 
-You will need to create a system user named `ccp_monitoring` which you can do with the following command:
+You will need to create a system user named {{< shell >}}ccp_monitoring{{< /shell >}} which you can do with the following command:
 
 ```bash
 sudo useradd -d /var/lib/ccp_monitoring ccp_monitoring
```
Comment (hunleyd) [inline hugo/content/prometheus/_index.md:49]: if you slap a `-m` on this, you can skip the 'you must create this dir' below

**Review by hunleyd — CHANGES_REQUESTED** [review]

File: hugo/content/exporter/_index.md [hunk 2]
```diff
@@ -144,75 +138,65 @@ First, make sure you have installed the PostgreSQL contrib modules.  You can ins
...
 | queries_bloat.yml     | postgres_exporter query file to allow bloat monitoring.                                                  |
-| queries_common.yml    | postgres_exporter query file with minimal recommended queries that are common across all PG versions.    |
+| queries_global.yml    | postgres_exporter query file with minimal recommended queries that are common across all PG versions and only need to be run once per database instance.    |
 | queries_per_db.yml    | postgres_exporter query file with queries that gather per databse stats. WARNING: If your database has many tables this can greatly increase the storage requirements for your prometheus database. If necessary, edit the query to only gather tables you are interested in statistics for. The "PostgreSQL Details" and the "CRUD Details" Dashboards use these statistics.                                                   |
-| queries_pg##.yml      | postgres_exporter query file for queries that are specific to the given version of PostgreSQL.           |
+| queries_general.yml      | postgres_exporter query file for queries that are specific to the given version of PostgreSQL where this file is installed.   |
```
[the remark this answers is not in the document]
Comment (keithf4) [reply hugo/content/exporter/_index.md:169]: Was meant to meant what version of PG is installed on the system. But multiple versions can also be installed. Reworded this and will push up shortly.

File: hugo/content/prometheus/_index.md (hunk 1 above)
Comment (keithf4) [reply hugo/content/prometheus/_index.md]: Updated this to add the -m. Where are you seeing a reference that says it needs to be created?

Comment (hunleyd) [reply hugo/content/prometheus/_index.md]: i was misreading the step below this and not seeing it was a subdir.

**Review by hunleyd — APPROVED** [review]

**Review by sharmay — DISMISSED** [review]

File: hugo/content/prometheus/_index.md [hunk 3]
```diff
@@ -84,48 +85,51 @@ The following pgMonitor configuration files should be placed according to the fo
...
-| `/etc/prometheus/alert-ruled.d/crunchy-alert-rules-\*.yml.example` | Update rules as needed and remove `.example` suffix. Prometheus config provided by pgmonitor expects `.yml` files to be located in `/etc/prometheus/alert-rules.d/` |
-| `/etc/prometheus/auto.d/*.yml` | You will need at least one file with a final `.yml` extension. Copy the example files to create as many additional targets as needed.  Ensure the configuration files you want to use do not end in `.yml.example` but only with `.yml`. Note that in order to use the provided Grafana dashboards, the extra "exp_type" label must be applied to the target and be set appropriately (pg or node). See the example target files provided for how to set the labels for postgres or node exporter targets. |
+| /etc/prometheus/crunchy-prometheus.yml | Modify to set scrape interval if different from the default of 30s. Activate alert rules and Alertmanager by uncommenting lines when set as needed. Activate blackbox_exporter monitoring if desired. Service file provided by pgMonitor expects config file to be named `crunchy-prometheus.yml` |
```
Comment (pgguru) [inline hugo/content/prometheus/_index.md:121]: Do the backticks in this (and others in this table) need to be converted to `shell` shortcodes, or they are fine in the table context?

Comment (sharmay) [reply hugo/content/prometheus/_index.md:121]: Our rules to not use decoration inside decoration, this mean no backticks inside table (table is decoration).

Comment (keithf4) [reply hugo/content/prometheus/_index.md:121]: Yeah I'd just removed the backticks all together here for that reason

Comment (pgguru) [reply hugo/content/prometheus/_index.md:121]: @keithf4 yeah, I'd noticed some in the replacement text (i.e.,: ``` `crunchy-prometheus.yml`  ```)

Comment (keithf4) [reply hugo/content/prometheus/_index.md]: Sorry, missed the ones in the description section. Fixed.

**Review by pgguru — APPROVED** [review]

**Review by hunleyd — APPROVED** [review]

**Pull request merged**